src/pudl/package_data/meta/datapkg/datapackage.json

```diff
@@ -5475,6 +5475,83 @@
                 "path": "https://creativecommons.org/licenses/by/4.0/"
             }],
             "sources": []
+        },
+        {
+            "profile": "tabular-data-resource",
+            "name": "assn_eia_epacems",
+            "path": "data/assn_eia_epacems.csv",
+            "title": "assn_eia_epacems",
+            "encoding": "utf-8",
+            "mediatype": "text/csv",
+            "format": "csv",
+            "dialect": {
+                "delimiter": ",",
+                "header": true,
+                "quoteChar": "\"",
+                "doubleQuote": true,
+                "lineTerminator": "\r\n",
+                "skipInitialSpace": true,
+                "caseSensitiveHeader": false
+            },
+            "schema": {
+                "fields": [{
+                        "name": "plant_id_eia",
+                        "type": "integer",
+                        "format": "default",
+                        "description": "The unique six-digit facility identification number, also called an ORISPL, assigned by the Energy Information Administration."
+                    },
+                    {
+                        "name": "plant_id_epa",
+                        "type": "integer",
+                        "format": "default"
+                    },
+                    {
+                        "name": "plant_name_eia",
+                        "type": "string",
+                        "format": "default"
+                    },
+                    {
+                        "name": "epa_point_source_unit",
+                        "type": "integer",
+                        "format": "default",
+                        "description": "'Smokestack' unit monitored by EPA CEMS."
+                    },
+                    {
+                        "name": "generator_id",
+                        "type": "integer",
+                        "format": "default"
+                    },
+                    {
+                        "name": "boiler_id",
+                        "type": "integer",
+                        "format": "default"
+                    },
+                    {
+                        "name": "prime_mover_code",
+                        "type": "string",
+                        "format": "default",
+                        "description": "Prime mover code as seen in EIA."
+                }],
+                "primaryKey": [
+                    "plant_id_eia"
+                ],
+                "foreignKeys": [{
+                    "fields": ["plant_id_eia", "generator_id"],
+                    "reference": {
+                        "resource": "generators_entity_eia",
+                        "fields": ["plant_id_eia", "generator_id"]
+                    }
+                }],
```

Comment:
The `primaryKey` has to uniquely identify a row in the table -- so unless there's only one record per `plant_id_eia`, I think we're going to need some additional columns in here. Pure association tables frequently include *all* of their columns as part of the primary key, since each row is meant to uniquely specify a relationship between other tables that are being referenced.  In a similar vein, you'll need at least 2 different `foreignKey` values in here, referring to the multiple different tables which are being related to each other through this association table.